Makefile

```diff
@@ -26,7 +26,7 @@ install:
 
 vendor-update:
 	@echo Updating vendored packages
-	@GO15VENDOREXPERIMENT=1 glide -q up
+	@GO15VENDOREXPERIMENT=1 glide install
```

Comment:
With glide-0.8.0, a new `glide.lock` file is created. `glide install` will install/update/downgrade packages listed in the lockfile.